README.md

```diff
@@ -97,6 +97,24 @@ BRANCHOUT_NAME=shorter-name
 
 to the Branchoutfile and ideally use the same value as the optionalDirName during branchout init.
 
+### Personal customisation
+
+Sometimes the branchout defaults don't work for a particular scenario even though they do work for most of your colleagues.
```

Comment:
see this is the problem... they should work for everyone or no one... there aren't really special cases for people although the org might be special

Reply:
Take a step back. What's important about this tool is *NOT* conformity and control of author over the world.

What's important about this tool is:

- Documenting a set of repos for a particular purpose within an organisation or for the public
- Efficiently managing large numbers of repos in a clean/easy way
- Consistent tooling for building projects with separated/isolated software repos
- Helper tooling for common things done on sets of projects (eg consistent git email and sshCommand config set on clone, may not be available yet, but I'm keen to add it)
- Consistent structure within a set of repos (currently only one style supported, but a few others have been brewing in my head for a while, keen to implement and use)

And all of that stuff is GREAT and a good reason for me to burn my personal time on this in the coming months, happy to become contributor number two to your project/great idea! :-D

Where on your disk those two roots are is NOT important, not even when you share a path to a colleague, unless they're of below average intelligence. The only part of the path that is key to consistency/understanding should be the BRANCHOUT_NAME within those two root directories, wherever they are. Typically paths I give to colleagues are in the context of the repo that the discussion is about, eg `src/main/java/` or whatever, not full absolute paths that may mean nothing on their disparate operating systems anyway. 

On my machine ~/ is cleared out every night at 4am against a whitelist so I had to update that whitelist to add two more entries and every time I `ls -al ~/` I see ~20% extra stuff that I don't want to. My personal structure after this is changed will be:

`~/repos/branchoutRoots`

and 

`~/repos/branchout/cacheAndSettingsRoots/`

then it'll all be in one place, nice and handy right next to my not-yet-managed-by-branchout structures that I'd *LOVE* to port over :-D 

I've told you before, I love this project. And I've also told you, it doesn't 100% work for me, yet. This is a step in the right direction that will also help many others with unpleasant usage situations to deal with get the best out of the tool in environments that they don't fully control.

Reply:
sounds reasonable